### PR TITLE
Fix README typos

### PR DESCRIPTION
**Pull request by JekRock**
- [x] tests and linter show no problems (`npm t`)
- [x] tests are added/updated for bug fixes and new features
- [x] code is properly formatted (`npm run fmt`)

Described steps aren't relevant to the changes because the changes are related to README only

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,23 +24,23 @@ development practices, and clean project structure.
 
 ### Metarhia and Impress application server way
 
-- Applied code need to be simple and secure, so we use code sandboxing with v8
+- Applied code needs to be simple and secure, so we use code sandboxing with v8
   isolated contexts, worker threads and minimal trusted npm dependencies;
-- Domain code need to be separated from system code; so we use DDE, layered
-  (onion) architecture, DI, SOLID:DIP principle and contract-based approach;
+- Domain code needs to be separated from system code; so we use DDE, layered
+  (onion) architecture, DI, SOLID, DIP principle and contract-based approach;
```
Comment (tshemsedinov) [inline README.md:30]:
```suggestion
  (onion) architecture, DI, SOLID:DIP principle and contract-based approach;
```

**Review by tshemsedinov — APPROVED** [review]
LGTM, except `SOLID:DIP`

File: README.md (hunk 1 above)
Comment (JekRock) [reply README.md]: Added suggested changes.
Sorry, didn't get it the first time. It's DIP from SOLID only, not "SOLID and DIP"

**Comment by tshemsedinov** [comment]
Steps are relevant to this PR and passed, because we have auto md formatter and README.md passed @JekRock

**Comment by tshemsedinov** [comment]
Landed in https://github.com/metarhia/impress/commit/14ee6862a645ecb93eda0fa7e1cf8573f00e781f

**Comment by tshemsedinov** [comment]
Thanks for contribution! @JekRock

**Comment by JekRock** [comment]
Glad to help!

**Pull request closed without merging**